src/linear_algebra/dimension.lean

```diff
@@ -153,6 +153,21 @@ letI := classical.dec_eq V₁; exact
 let ⟨b, hb⟩ := exists_is_basis K V in
 cardinal.lift_inj.1 $ hb.mk_eq_dim.symm.trans (f.is_basis hb).mk_eq_dim
 
+/-- Two vector spaces are isomorphic if they have the same dimension. -/
+theorem nonempty_linear_equiv_of_dim_eq (cond : dim K V = dim K V₁) :
+  nonempty (V ≃ₗ[K] V₁) :=
+begin
+  obtain ⟨B,h⟩ := exists_is_basis K V,
+  obtain ⟨B₁,h₁⟩ := exists_is_basis K V₁,
+  have : cardinal.mk B = cardinal.mk B₁, by rwa [is_basis.mk_eq_dim'' h, is_basis.mk_eq_dim'' h₁],
+  rw cardinal.eq at this,
+  exact nonempty.map (equiv_of_is_basis h h₁) this,
```

Comment:
Could you take this opportunity to rename `equiv_of_is_basis` to `linear_equiv_of_is_basis`?

Reply:
sure :)